test/integration/test_resolvers.py

```diff
@@ -0,0 +1,24 @@
+"""Integration tests for the conda dependency resolution."""
+
+import os
+from base import integration_util
+from base.api import ApiTestCase
+
+GNUPLOT = {u'version': u'4.6', u'type': u'package', u'name': u'gnuplot'}
+
+class CondaResolutionIntegrationTestCase(integration_util.IntegrationTestCase, ApiTestCase):
```

Comment:
Thanks a lot @jmchilton, importing svgwrite after from base import integration_util works.
That's good enough for a workaround, right?